### PR TITLE
rename JeOS to Minimal and remove Vagrant images

### PR DESCRIPTION
**Pull request by suntorytimed**

### CODE REVIEW

File: gocd/sp4.target.gocd.yaml [hunk 1]
```diff
@@ -151,7 +152,8 @@ pipelines:
             tasks:
               - script: |-
                   PRODUCTS=$(osc -A https://api.suse.de ls SUSE:SLE-15-SP4:GA | grep "^SLES15-SP4" | grep -v release)
-                  for product in kiwi-templates-JeOS SLES-15-SP4-Vagrant sles15-image 000product $PRODUCTS; do
+                  set -e
```
Comment (Vogtinator) [inline gocd/sp4.target.gocd.yaml:155]: Move before the `osc` call above?

File: gocd/sp4.target.gocd.yaml [hunk 2]
```diff
@@ -82,7 +82,8 @@ pipelines:
         tasks:
         - script: |-
             PRODUCTS=$(osc -A https://api.suse.de ls SUSE:SLE-15-SP4:GA | grep "^SLES15-SP4" | grep -v release)
-            for product in kiwi-templates-JeOS SLES-15-SP4-Vagrant sles15-image 000product $PRODUCTS; do
+            set -e
```
Comment (Vogtinator) [inline gocd/sp4.target.gocd.yaml:85]: Move before the `osc` call above?

Comment (suntorytimed) [reply gocd/sp4.target.gocd.yaml:85]: Done 👍

File: gocd/sp4.target.gocd.yaml (hunk 1 above)
Comment (suntorytimed) [reply gocd/sp4.target.gocd.yaml:155]: Done 👍

File: gocd/sp4.target.gocd.yaml (hunk 2 above)
Comment (Vogtinator) [reply gocd/sp4.target.gocd.yaml:85]: You might've forgotten to `git push`

Comment (suntorytimed) [reply gocd/sp4.target.gocd.yaml:84]: Even stupider. I forgot to add before amending ;)

**Review by Vogtinator — APPROVED** [review]

**Pull request merged**